### ISSUE TITLE
Integrate custom sampler orders for HF transformers loaders

### ISSUE THREAD

**Issue by kalomaze** [oobabooga/text-generation-webui#4463]
I think this is extremely important to have as a feature. I've tested both Temperature first and Temperature last in Koboldcpp, and temp first seems inferior to me. This is because you are unable to control the randomness of the logits *after* samplers (like Min P) are performed, if you are setting temp last. Truncation based samplers measure in a way that is **relative**; this means that tokens are not being measured in a way that is consistent before temperature scaling is applied. For example, if you have a Min P of 0.05, and your original top token is 95% followed by a 0.5% token, this would not normally be selected by Min P for consideration, even if temperature was maxed out. But since Min P comes _afterwards_, it's possible for a high enough temp to allow for that low scoring token when it normally would not have included it at all.

**Comment by kalomaze** [comment]
https://github.com/oobabooga/text-generation-webui/pull/4472 accounts for this issue
